sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java

```diff
@@ -937,6 +940,16 @@ public CreateNewCaseSteps(
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(POINT_OF_ENTRY_REGION_BUTTON);
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(POINT_OF_ENTRY_DISTRICT_BUTTON);
         });
+
+    When(
+        "I check if collected case UUID is equal with current",
+        () -> {
+          softly.assertEquals(
+              casesUUID.get(0),
```

Comment:
why we need a list if we always collect one value?

Reply:
This list is used in some previous test. I reused it without create a new variable to that.